ci/rebase_analysis.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
Since `fedora-np` node is being used, maybe it would be better to say python2 here. That will ensure Python 2 and will be noisy if Python 2 is not available at some point.